app/models/conversation.rb

```diff
@@ -207,6 +207,13 @@ def csat_survey_link
     "#{ENV.fetch('FRONTEND_URL', nil)}/survey/responses/#{uuid}"
   end
 
+  def push_sla_change_event
```

Comment:
let's make this something more generic  like
```suggestion
  def dispatch_conversation_updated_event(previous_changes=nil)
     dispatcher_dispatch(CONVERSATION_UPDATED, previous_changes)
  end
```
replace the line 245 with this method as well.

Reply:
@sojan-official Done https://github.com/chatwoot/chatwoot/pull/9146/commits/1018b70afcaa9936164fced41faa94e454f9de8f